pages/lib/refinery/pages/configuration.rb

```diff
@@ -26,6 +26,15 @@ class << self
       def layout_template_whitelist
         Array(config.layout_template_whitelist).map(&:to_s)
       end
+
+      def default_parts
+        if config.default_parts.all? { |v| v.is_a? String }
+          Refinery.deprecate('Changes in Refinery::Pages.default_parts config', when: '3.2')
```

Comment:
This doesn't really tell me how to fix it.  Given Refinery knows what I've specified, it can suggest it for me:

```ruby
Refinery.deprecate("Change Refinery::Pages.default_parts= from #{config.default_parts.inspect} to #{config.default_parts.map{|v| {title: v, slug: v.downcase.gsub(" ", "_")} }}", when: "3.2.0")
```

or something?